.simpleversion.json

```diff
@@ -1,6 +1,5 @@
 {
-  "version": "0.1.0",
-  "label": [ "alpha2" ],
+  "version": "0.1.*",
```

Comment:
Not quite ready for this to be bumped to a release branch.  This will need to be reverted.